### ISSUE TITLE
Not working anymore after 16.06.2020 update?

### ISSUE THREAD

**Issue by Flow86** [alinmear/docker-conanexiles#38]
Hi,

it's not directly a issue of the docker container itself, but
is it possible that running conan exiles server via wine is not working anymore via wine? does someone has an idea how to fix it?

It always crashes somewhere

**Comment by Flow86** [comment]
Ah after adding the comodo certificate it works again:

```
ADD http://crt.usertrust.com/USERTrustRSAAddTrustCA.crt /usr/local/share/ca-certificates/USERTrustRSAAddTrustCA.crt
RUN update-ca-certificates
```

**Comment by matthewmcpartland** [comment]
Did you add the cert to the redis instance? I currently only run a single conan instance without a redis, I added the cert and ran `update-ca-certificates` and restarted the docker compose I have, the cert persisted but the issue hasn't been resolved for me.

When not running the docker detached, I do not see any outputs indicating the server is crashing, however I'm still not seeing the server listed in the server browser and I'm not able to connect directly via IP over intranet.
The network errors I receive: 
```
conanexiles_1  | Network:Error: Code: Certificate path env var 'DW_CERTIFICATE_PATH' is not set
conanexiles_1  | Network:Warning: Code: Loading dedicated server provider id certificate: SecureServerProviderID not set.
conanexiles_1  | Network:Display: Looking for dedicated server provider id certificates issued by "C=NO, S=Oslo, L=Oslo, O=Funcom Oslo AS, CN=Funcom Oslo AS"...
conanexiles_1  | Network:Warning: Code: Loading dedicated server provider id certificate: No secure server provider id configured, selecting certificate with id #-1.
conanexiles_1  | Network:Error: Code: KDS passphrase provider failed to initialize. KDS will at some point deny access unless the problem is fixed. Failure message: Error loading dedicated server provider id certificate: No certificate found with id #-1..
conanexiles_1  | Network:Error: Code: KDS passphrase failure: Server provider id certificate unavailable: 0x80092004
conanexiles_1  | Network:Display: Connecting to key distribution web service wss://ce-kds-winunoff-wdc05.funcom.com:7004/server/...
conanexiles_1  | Network:Display: Key distribution web service: Connecting (session: 831F5E454D51D2B1E7A26E856477C3F3, mirror: wss://ce-kds-winunoff-wdc05.funcom.com:7004/server/, fail count: 0, retry time: -1.0)
```

**Comment by alinmear** [comment]
I guess you guys still using the very old docker image:

```
alinmear/docker-conanexiles:_12
Digest:sha256:0bb535b461d5428cd8b88aa16f242f75517f2302991addadc23da346ea34675e
```

If so, this image is 2 years old; does this also happen with :latest ?

**Comment by Flow86** [comment]
The certificate stuff also is a problem on "latest" - I build my image from source, since I've added extensions for automatic mod support to it. But I can't speak for matthew

**Comment by Flow86** [comment]
you can look into my changes here: You can freely use anything from it:

https://github.com/alinmear/docker-conanexiles/compare/master...Flow86:master

**Comment by alinmear** [comment]
Interesting, just made a fresh local provisioning with my latest builds and there are no problems with the certificates. exiles server is installed without problems via steam.

Thanks for sharing your work. i look into your changes, so we can merge it to have proper synergies.

**Comment by alinmear** [comment]
I just merged your changes @Flow86, thx for your work; maybe we can move parts into the Dockerfile; waiting for your comment

**Comment by alinmear** [comment]
Fixed with #40
